crates/rate-limiter/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod rate_limiter;
+
+pub use rate_limiter::{RateLimiter, RateLimiterError, RateLimitingStrategy};
```

Comment:
Wondering if `Error` and `Strategy` should just be called that (we can label them `rate_limiter::Strategy` where the context is needed)?